lms/djangoapps/certificates/admin.py

```diff
@@ -57,9 +58,18 @@ class CertificateTemplateAssetAdmin(admin.ModelAdmin):
     """
     Django admin customizations for CertificateTemplateAsset model
     """
+
     list_display = ('description', 'asset_slug',)
     prepopulated_fields = {"asset_slug": ("description",)}
 
+    # see PROD-1153 for the details
+    def changelist_view(self, request, extra_context=None):
+        if '.stage.edx.org' in request.get_host():
+            extra_context = {'title': mark_safe('Select Certificate Template Asset to change <br/><br/>'
+                                                '<div><strong style="color: red;"> Warning!</strong> Updating '
+                                                'stage asset would also update production asset</div>')}
+        return super(CertificateTemplateAssetAdmin, self).changelist_view(request, extra_context=extra_context)
+
 
```

Comment:
I think the above text should be multi lingual. Please add code to make it run for other languages too.